build-config/checkstyle/build.gradle

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
 * Copyright (c) 2022, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Reply:
Updated in [7c05127](https://github.com/ballerina-platform/protoc-tools/pull/3/commits/7c05127715f921c585ed47b679cc1c179d5610f8)